### ISSUE TITLE
 Could not find method getCompileConfiguration()

### ISSUE THREAD

**Issue by muehan** [EddyVerbruggen/nativescript-plugin-firebase#715]
Im Only using Android

**Version**
```
"nativescript-angular": "~5.3.0",
"nativescript-plugin-firebase": "^6.0.2",
```

**What I have done:**

1. run my app and it worked as it should
2. tns plugin add nativescript-plugin-firebase
3. create firebase app and download google-services.json to App_Resources/Android
4. delete npm_modules and platfroms
5. npm install
6. tns platform add andoird
7. tns run android

Error:

```
Building project...
Gradle build...
         + applying user-defined configuration from D:\git\ELD-Mobile\app\App_Resources\Android\app.gradle
         + add include.gradle from D:\git\ELD-Mobile\node_modules\nativescript-plugin-firebase\platforms\android\include.gradle
         + applying plugin configuration from D:\git\ELD-Mobile\platforms\android\app\build\configurations\nativescript-plugin-firebase\include.gradle
         + adding nativescript runtime package dependency: nativescript-optimized
         + adding aar plugin dependency: D:\git\ELD-Mobile\node_modules\nativescript-plugin-firebase\platforms\android\firebase-release.aar
         + adding aar plugin dependency: D:\git\ELD-Mobile\node_modules\tns-core-modules-widgets\platforms\android\widgets-release.aar

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not find method getCompileConfiguration() for arguments [] on object of type com.android.build.gradle.internal.api.ApplicationVariantImpl.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org
```

**What have I tried:**
1.  update nativescript => now version 4.0.2
2. delete and reinstall npm_modules and platfroms
3. googling the error message and out of that:
=> update classpath 'com.android.tools.build:gradle:3.1.0'
which then needs a newer gradle verision
=> updated to distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip

the only result I have so far is another error message.

**Comment by EddyVerbruggen** [comment]
That’s new for me. Did you try the demo app(s) in this repo?

**Comment by elimin8r** [comment]
+1

**Comment by 7ammer** [comment]
getting the same issue.

**Comment by andreMariano90** [comment]
fix here -> #722

**Comment by muehan** [comment]
I fixed it by updating everything.

```
npm install -g nativescript
tns update

```

deleted package-lock.json
deleted node_modules
deleted platfroms

npm install tns-core-modules --save (new dependency)
npm install
tns platform add android

now it compiles
